Runmon/47_Sirius.yaml

```diff
@@ -0,0 +1,64 @@
+species: Breloom
+ingamename: "'HH   "
+displayname: "'HH   "
+setname: TPP Sirius
+tags: [runmon, in-game]
+item: [Leftovers]
+ability: [Effect Spore]
+moves:
+    - [Focus Punch]
+    - [Leech Seed]
+    - [Substitute]
+    - [Spore (=18)]
```

Comment:
```suggestion
    - [Spore (+1)]
```

Reply:
Well, you said yourself in the linked issue that Hippopotas should be included too, and Wobbuffet was HoFed, even if it didn't battle much.